### PR TITLE
.Net: Fix Solution File

### LINKED ISSUE #2975: [Cleanup] Make sure solution build configurations are accurate debug|release|publish

### PR DESCRIPTION
**Pull request by hario90**
### Motivation and Context

Fixes #2975

The Planners.Core.UnitTests project was missing a EndProject tag (Thanks @lemillermicrosoft for catching this!). Additionally, the mapping for the publish profile was incorrect for the Planners.Core project.

### Contribution Checklist

- [x] The code builds clean without any errors or warnings
- [x] The PR follows the [SK Contribution Guidelines](https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md) and the [pre-submission formatting script](https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md#development-scripts) raises no violations
- [x] All unit tests pass, and I have added new tests where possible
- [x] I didn't break anyone :smile:

### CODE REVIEW

File: dotnet/SK-dotnet.sln [hunk 1]
```diff
@@ -161,6 +161,7 @@ EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Planners.Core", "src\Planners\Planners.Core\Planners.Core.csproj", "{F224B869-FA0E-4EEE-A6BF-C2D61FF8E731}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Planners.Core.UnitTests", "src\Planners\Planners.Core.UnitTests\Planners.Core.UnitTests.csproj", "{CC77DCFA-A419-4202-A98A-868CDF457792}"
+EndProject
```
Comment (lemillermicrosoft) [inline dotnet/SK-dotnet.sln:164]: Want to remove the duplicate `{3CDE10B2-AE8F-4FC4-8D55-92D4AD32E144} = {958AD708-F048-4FAF-94ED-D2F2B92748B9}` below as well? I saw this today.

Comment (hario90) [reply dotnet/SK-dotnet.sln:164]: Fixed. Good catch!

**Review by markwallace-microsoft — APPROVED** [review]

**Review by dmytrostruk — APPROVED** [review]

**Pull request merged**